docs/2.9.0/docs/canton/usermanual/FAQ.rst

```diff
@@ -163,6 +163,13 @@ system:
 
 .. literalinclude:: /canton/includes/mirrored/community/app/src/test/resources/documentation-snippets/dev-version-support.conf
 
+How do I enable preview features?
+---------------------------------
+For early access to new features, enable preview version support.
```

Comment:
I would expand a bit here. We need to say at least:

- That a preview protocol version is a protocol version that allows to use features that have initial availability.
- That we offer support for IA features only if there is an agreement for that. So basically, a customer using Daml enterprise can get support for IA features only if we agreed with them to support this use case in prod.